ls1intum/Apollon_standalone
# ``Development``: Enhance Share Modal of Apollon standalone

### Motivation and Context

This PR enhances the currently existing share modal of Apollon Standalone. 
The current version of the Share modal requires 4 clicks to be done before sharing a link, which is a time-consuming task.
This proposed share modal reduces the number of clicks to 2. 

### Steps for Testing
To test locally:
1. Clone the repo
2. checkout the `enhancement/change-share-modal`  branch
3. ` yarn install && yarn start`
4. Once the application is running, go to the share menu and observe the newly implemented buttons instead of the old dropdown menu entries.

To test in test server: 
1. Goto https://test1.apollon.ase.in.tum.de
 
### Screenshot:
Before:
![screen-capture](https://user-images.githubusercontent.com/14681902/168789759-2adea539-6c5e-4d4c-acee-7f36fa15a1a7.gif)


After:
![screen-capture-_1_](https://user-images.githubusercontent.com/14681902/168789811-1bb8a3dd-2b1f-475f-a2b2-2df146e2a076.gif)


### Todo:

- [x] Deploy in test server
